src/types/integrations.ts

```diff
@@ -9,6 +9,11 @@ import { ChipItem } from '@zextras/carbonio-design-system';
 
 import { Contact } from './contact';
 
+export const CHIP_DISPLAY_NAME_VALUES = {
```

Comment:
This const should not be here. The meaning for this file, as I understand from the structure, is to be a declaration file with only types (it could be a .d.ts file). A const is not a type, therefore, it should be placed somewhere else (inside a constants file or near its usage)

Reply:
Should I create a constant directory and create a new file there? in contacts I didn't find a useful place to group all the constants

Reply:
imo you can either create a constants directory, or put it inside the contact-input-custom-chip-component since it is strictly related to that component

Reply:
This constant is used inside 3 files:
- Contact Input
- Contact input custom component
- Contact input types file
I wanted to keep it inside the Contact input custom component but if I place it there it would create an import dependency cycle. Same if I put it inside the Contact Input. That is why I mistakenly placed it inside the types file. I didn't want to create another file for this constant as it is strictly related to these 3 files. To me looked like the best solution between all.